test/e2e-external/setup_external_resources.go

```diff
@@ -50,12 +52,17 @@ func SetupTestEnv(k8sClient client.Client, namespace string) (TestEnv, error) {
 	}
 	cleanupFuncs = append(cleanupFuncs, cleanupPostgres)
 
-	cleanupMinio, err := CreateFromFile(ctx, k8sClient, namespace, "../../resources/minio.yaml")
-	if err != nil {
-		cleanupFuncs.Cleanup()
-		return TestEnv{}, errors.Wrap(err, "failed to apply Minio")
+	cmd := exec.Command("kubectl", "minio", "init")
```

Comment:
Same. How do we ensure that `minio` plugin is installed in `kubectl`?

Reply:
It's all included in `make mysql-minio-operators` (scripts/install-mysql-minio.sh).